cypress/support/config/settings.js

```diff
@@ -6402,21 +6437,8 @@ module.exports = () => ({
         smoke: false,
       },
       idxPage: {
-        environments: {
-          live: {
-            paths: ['/ukrainian/ukraine_in_russian'],
-            enabled: false,
-          },
-          test: {
-            paths: ['/ukrainian/ukraine_in_russian'],
-            enabled: true,
-          },
-          local: {
-            paths: ['/ukrainian/ukraine_in_russian'],
-            enabled: true,
-          },
-        },
-        smoke: true,
+        environments: undefined,
+        smoke: false,
       },
```

Comment:
Our cypress set up is tightly coupled so that each service has to declare the page types, otherwise, it would error. Note we are passing `idxPage: { environments: undefined, smoke: false }` in other services that do not have this page type as well

![image](https://user-images.githubusercontent.com/30599794/85531287-8a071380-b606-11ea-8c95-6f183a930675.png)